Source/WebCore/page/Quirks.cpp

```diff
@@ -3166,6 +3164,15 @@ static void handlePandoraQuirks(QuirksData& quirksData, const URL& /* quirksURL
     quirksData.enableQuirk(QuirksData::SiteSpecificQuirk::ShouldExposeShowModalDialog);
 }
 
+static void handlePinterestQuirks(QuirksData& quirksData, const URL& /* quirksURL */, const String& quirksDomainString, const URL&  /* documentURL */)
+{
+    if (quirksDomainString != "pinterest.com"_s) [[unlikely]]
+        return;
+
+    // pinterest.com rdar://104979314
```

Comment:
Can you also add a comment that this quirk should be removed when rdar://165745719 is resolved?